scripts/scripts.js

```diff
@@ -34,13 +34,46 @@ function buildHeroBlock(main) {
   }
 }
 
+function buildTextMediaBlock(main) {
+  // Find blocks that contain a picture followed by an em text block. These are text-media blocks
```

Comment:
You may want to check that this works well when images are themselves present inside other blocks and have italicised text following them.  For e.g. the carousel block.

Reply:
Yes I have checked this.